mne/stats/regression.py

```diff
@@ -18,6 +19,14 @@
 from ..io.pick import pick_types
 
 
+def get_fast_dot():
+    try:
+        from sklearn.utils.extmath import fast_dot
+    except ImportError:
+        fast_dot = np.dot
+    return fast_dot
```

Comment:
don't we already have this elsewhere? please import from wherever that is, or better move it to e.g. `mne/fixes.py`